src/app/modules/auth/store/auth.actions.ts

```diff
@@ -1,6 +1,6 @@
-// tslint:disable: max-line-length
 import { createAction, props } from '@ngrx/store';
 
+// tslint:disable: max-line-length
```

Comment:
full file disables should be in the start of the file revert moving this file down